backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Tests/Controllers/ResourceControllerTest.cs

```diff
@@ -233,6 +242,30 @@ public async Task GetSingleRightsSearch_noSearchStringAndFilters_ValidPagination
             }
         }
 
+        /// <summary>
+        ///     Test case: GetAllResourceOwners, returns a list of resource owners in string format
+        ///     Expected: GetResources returns a list of resource owners with language filtered for the authenticated users
+        ///     selected
+        /// </summary>
+        [Fact]
+        public async Task GetAllResourceOwners_validresults()
+        {
+            // Arrange
+            OrgList orgList = ResourceUtil.GetMockedResourceRegistryOrgList();
+            List<ResourceOwnerFE> expectedResult = _resourceService.MapOrgListToResourceOwnerFe(orgList, "nb");
```

Comment:
I don't know if I like the direct use of the resourceService here.

I get that you don't want the duplicate code, but a test that checks the result of a some code by using that very same code to define what is expected... that doesn't sit right with me. If there is a bug in the reused mapping, this test won't be able to discover it.

Reply:
I understand what you mean but in some way this is what will be done in some way? Yeah I  could instead copy the mocked data from the json and deserialize the json, but that's just then I just do the same with more code duplicate. I think testing mocked clients is a bit silly.

I agree that we don't really test anything here, if the test didn't went through the real client and not through the mocked clients it would give value though. But I really think we should just delete the whole test and not bother to test mocked clients.

Reply:
Ahh now I think I understand what you mean. You just mean that I should create a static method that only creates the expected result => a static method that only returns the full OrgList?

Reply:
I fixed